README.md

```diff
@@ -25,35 +31,41 @@ $ curl http://localhost:8081/<report_name>?<params>
 
 ## API
 
-### URL
+### URLs
 
-`/<string:report_name>?<params>` or `/report/<string:report_name>?<params>`
-
-HTML table representation of report at `/table/<string:report_name>?<params>`
+`/v1/<string:report_name>/?<params>`
+`/v1/<string:query_type>/<string:report_name>/?<params>`
+`/v1/sql/<string:report_name>/<string:format>/?<params>`
+`/v1/sql/<string:report_name>/compact/?<params>`
 
 `report_name` should match the name of a given SQL file, minus the `.sql` extension. Params are as specified in the queries themselves.
```

Comment:
Should this be written to also be applicable to other types of local files? This seems fine, just thought it would be worth checking.

Reply:
Right. Good catch. This line didn’t get updated with the refactor